packages/core/src/components/InlineEditor/InlineEditor.tsx

```diff
@@ -82,8 +82,12 @@ export const HvInlineEditor = ({
   const [editMode, setEditMode] = useState(false);
   const [cachedValue, setCachedValue] = useState(value);
   const inputRef = useRef<HTMLInputElement>();
+  const { rootId } = useTheme();
 
-  const typographyStyles = getTypographyStyles(theme.typography[variant] || {});
+  const typographyStyles = getTypographyStyles(
+    theme.typography[variant] || {},
+    rootId
+  );
```

Comment:
Why not use the object value?
```suggestion
  const typographyStyles = activeTheme?.typography[variant] || {};
```